classes-metodos-atributos/exercicio-05-06/src/main/java/MyProgram.java

```diff
@@ -0,0 +1,48 @@
+import java.util.Random;
+import java.util.Scanner;
+
+public class MyProgram {
+
+    Scanner in = new Scanner(System.in);
```

Comment:
Verificar se é necessário fechar o in.

Reply:
Essa é a mensagem que está na aba PROBLEMAS/PROBLEMS do VSCode: Resource leak: 'in' is never closed.
Então, sim, precisa ser fechada e o código precisa vir sem nenhum PROBLEMA relatado nessa aba. 
Faça as verificações antes de enviar.

Reply:
Não precisa fechar o "in", está correto a chamada da função Scanner in = new Scanner (System.in).

Reply:
Para solucionar o erro do in, na função Scanner, inserir in.close();